ui/admin/tests/acceptance/aliases/delete-test.js

```diff
@@ -17,7 +17,8 @@ module('Acceptance | aliases | delete', function (hooks) {
 
   let aliasCount;
 
-  const DROPDOWN_BUTTON_SELECTOR = '.hds-dropdown-toggle-icon';
+  const DROPDOWN_BUTTON_SELECTOR =
```

Comment:
None blocking comment: since this PR is not covering test selectors refactor, and this specific domain is not yet finished on the refactor I am fine reusing present selectors regardless name or content 😉